generic_tracers/generic_COBALT.F90

```diff
@@ -3596,19 +3640,21 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,rho_dzt,dzt,h
 
     enddo; enddo; enddo  !} i,j,k
 
+    ! Assume that individually sinking phytoplankton, which sink at slow rates relative to aggregates and fecal
+    ! pellets, collect in a nepholoid layer and are available for resuspension if they are exposed to mixing. This
+    ! is accomplished by setting the vertical sinking rate in the bottom layer to 0, and is assumed to occur when 
+    ! when the depth is less than twice the depth of active mixing.  Cells are otherwise assumed to sink into the
+    ! benthos and be remineralized along with sinking detritus.
+
     do j = jsc, jec ; do i = isc, iec   !{
-       !
-       ! assume that individually sinking phytoplankton collect in nepholoid layer
-       ! and are available for resuspension if they are exposed to mixing
        do n = 1,NUM_PHYTO
          if (cobalt%zt(i,j,nk).le.(2.0*hblt_depth(i,j))) then
```

Comment:
I'm curious about this -- if you use just the `hblt_depth` here instead of twice the depth of active mixing, does this result in too much burial and removal of phytoplankton N/Si/Fe?

Reply:
Hi Jessica - right now burial is turned off since we were very concerned with too much burial in regional applications with enhanced resolution of nearshore areas.   We also found, however, that assuming that slowly sinking cells were immediately buried and remineralized in the nearshore regions rather than being available for re-entrainment was suppressing nearshore chlorophyll.  

I think these will be a good topic for additional sensitivity, with the Mississippi hypoxic zone likely providing a good test case.